codebuild/codebuild.config

```diff
@@ -61,12 +73,12 @@ env: S2N_LIBCRYPTO=openssl-1.1.1 OPENSSL_ia32cap="~0x200000200000000" BUILD_S2N=
 
 # Unit Test
 [CodeBuild:s2nUnitOpenSSL111Gcc6]
-snippet: UbuntuBoilerplate2XL
+snippet: UbuntuBoilerplateLarge
 env: S2N_LIBCRYPTO=openssl-1.1.1 BUILD_S2N=true TESTS=unit GCC_VERSION=6 S2N_COVERAGE=true CODECOV_IO_UPLOAD=true
 
 # Asan Test
 [CodeBuild:s2nAsanOpenSSL102Gcc6]
-snippet: UbuntuBoilerplate2XL
+snippet: UbuntuBoilerplateLarge
```

Comment:
What are plans for changing other jobs over? Should other stakeholders examine/list their requirements?

Reply:
Good call out.  I added a section in the Readme about our experience to date on this topic.